draft-ietf-quic-qlog-main-schema.md

```diff
@@ -244,6 +246,57 @@ schema structured as:
     * `scenario` event type
     * `marker` event type
 
+# Abstract LogFile Class {#abstract-logfile}
+
+A Log file is intended to contain a collection of events that are in some way
+related. An abstract LogFile class containing fields common to all log files is
+defined. Each concrete log file format derives from this, extending it by
+defining semantics and any custom fields.
+
+~~~ cddl
+LogFile = {
+    file_schema: text
+    qlog_serialization_format: text
+    ? title: text
+    ? description: text
+    event_schema: [+text]
```

Comment:
should this be `event_schemas` instead (making it implicitly clear there is 1 `file_schema` but multiple `event_schemaS`?

Reply:
yep, done